test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -36,7 +36,7 @@ func SkipTestIfLocal(t *testing.T, msg string, f func(t *testing.T)) {
 func StatefulSetBecomesReady(mdb *mdbv1.MongoDBCommunity, opts ...wait.Configuration) func(t *testing.T) {
 	defaultOpts := []wait.Configuration{
 		wait.RetryInterval(time.Second * 15),
-		wait.Timeout(time.Minute * 15),
+		wait.Timeout(time.Minute * 20),
```

Comment:
[q] I know you changed this a few times for testing, is it needed now?

Reply:
ah nice catch, right we shouldn't need this anymore.